### PR TITLE
api: add healthz route

### LINKED ISSUE #196: implement api similar to healthz to verfiy liveness and readiness of zot server

### PR DESCRIPTION
**Pull request by shimish2**
Added /v2/healthz route to verify liveness and readiness probe of zot server

Successful request to /v2/healthz route will return status code 200

Closes #196

### CODE REVIEW

File: pkg/api/routes.go [hunk 1]
```diff
@@ -96,6 +98,17 @@ func (rh *RouteHandler) SetupRoutes() {
...
 
+// CheckHealth godoc
+// @Summary Check Health API to check server health
+// @Description Check if zot server is up or not
+// @Router /v2/ [get]
+// @Accept json
+// @Accept json
+// @Success 200 {string} string "ok".
+func (rh *RouteHandler) CheckHealth(w http.ResponseWriter, r *http.Request) {
+	WriteData(w, http.StatusOK, "application/json", []byte{})
```
Comment (rchamarthy) [inline pkg/api/routes.go:109]: what is the actual health check here? Just that http router and the port is working?

Comment (shimish2) [reply pkg/api/routes.go:109]: Yes

Comment (shimish2) [reply pkg/api/routes.go:109]: Do you want me to add more ?

Comment (mikemccracken) [reply pkg/api/routes.go:109]: I think this is enough for starters, if we can't respond to this then an orchestrator should definitely restart us.

**Pull request closed without merging**